### DISCUSSION TITLE
Xiaomi Hub 2 add

### DISCUSSION THREAD

**Discussion by Strawqens** [merdok/homebridge-miot#541] [Q&A]
Some time ago, Xiaomi released a new hub, Xiaomi Hub 2, which no longer supports Apple Homekit (in my opinion, it's a step backward). Will it be included in the miot plugin? Or is there a way to add it already?

**Comment by merdok** [comment]
It should be already possible to do so. You can try to add and let me know if there are any issues.

**Reply by Strawqens** [reply]
Instead of displaying as a Gateway with connected devices, it shows something like this, and no function is working
![Uploading Zrzut ekranu 2023-11-2 o 18.10.25.png…]()

**Reply by merdok** [reply]
I cannot open the image which you provided, but you will not able to see a list of connected devices using the miot protocol. The hub does not report that. I highly recommended that you visit the page https://www.merdok.org/miotspec/ and check what your device supports.
